src/database/schema/ChannelListSchema.ts

```diff
@@ -62,6 +64,7 @@ class ChannelList implements BaseDbSchema {
     user,
     expiredAt = null,
     members = [],
+    memberUsers = [],
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The constructor has been updated to include `memberUsers` as a parameter. This is consistent with the new property added to the class.
```diff
+    memberUsers = [],
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->